js/classes/zone.js

```diff
@@ -0,0 +1,141 @@
+            // Clear all entities from previous Zone.
+            GAME.clearEntities();
+            // Clear all images/music from previous Zone.
+            ASSET_MGR.clearCache();
+            loadFunction();
+        };
+    };
+
+    // DYNAMIC GETTERS:
+
+    /**
+     * @returns {Vector} The Min (upper left) block that is visible in this Zone.
+     */
+    get MIN_BLOCK() {
+        return this.minBlock;
+    };
+
+    /**
+     * @returns {Vector} The Max (bottom right) block that is visible in this Zone.
+     */
+    get MAX_BLOCK() {
+        return this.maxBlock;
+    };
+
+    /**
+     * @returns {Vector} The minimum (x, y) coordinate visible in this Zone.
+     */
+    get MIN_PT() {
+        return Vector.multiply(this.MIN_BLOCK, Block.SCALED_SIZE);
+    };
+
+    /**
+     * @returns {Vector} The maximum (x, y) coordinate visible in this Zone.
+     */
+    get MAX_PT() {
+        // I want to actually be able to SEE the final block (not just touch its top left corner). 
+        // Therefore:
+        const ones = new Vector(1, 1);
+        return Vector.multiply(Vector.add(this.MAX_BLOCK, ones), Block.SCALED_SIZE);
+    };
+
+    // I am not sure if we are actually going to need the following two getters, but I include them just in case:
+
+    /**
+     * @returns {Vector} The size of the zone, in BLOCKS.
+     */
+    get BLOCK_SIZE() {
+        // We gotta include an extra block to account for 0.
+        const ones = new Vector(1, 1);
+        return Vector.add(new Vector(this.MAX_BLOCK.x - this.MIN_BLOCK.x, this.MAX_BLOCK.y - this.MIN_BLOCK.y), ones);
+    };
+
+    /**
+     * @returns {Vector} The size of the zone, in PIXELS.
+     */
+    get PIXEL_SIZE() {
+        return Vector.multiply(this.BLOCK_SIZE, Block.SCALED_SIZE);
+    };
+
+    /**
+     * This is the only way which we should be externally generating Zones! Navigate this object to set ZONE to the one you want,
+     * and then call its load() function in order to get everything properly loaded for your new Zone.
+     * @returns An object containing ALL ZONES!
+     */
+    static getZones() {
+        // Right now, I am creating zones with no real idea how big they ought to be/what ought be in them.
+        // Therefore, I am defining these constants for MIN/MAX_BLOCK.
+        const zeros = new Vector(0, 0);
+        const defaultMaxBlock = new Vector(100, 25);
+
+
+        /*
+            NOTE: EVERYBODY WILL BE PUTTING CODE HERE! PLEASE:
+
+            (A)     Keep it sorted appropriately. There should only ever be the seven dimensions - playground + the six in-game ones.
+            (B)     Keep it alphabetized.
+            (C)     Match the naming style for your load method! load<Dimension><ZoneName>
+            (D)     Add as many zones to the playground as you want! Avoid adding temporary Zones to real Dimensions.
+            (E)     Be sure that your load function is defined in the proper script and imported into the head of index.html.
```

Comment:
We need to import a function? Not a script? @B1GD3VD0G

Reply:
The function is defined in the script. The script is imported into the head of the document. It would be a good idea for one of us to change the phrasing a little - I'll try and remember to change it before the next PR

Reply:
Sounds good! That makes more sense. Thank you.